src/desktop/apps/consign/client/analytics_middleware.js

```diff
@@ -21,7 +22,11 @@ const analyticsMiddleware = store => next => action => {
     }
     case actions.SUBMISSION_CREATED: {
       analyticsHooks.trigger("consignment:submitted", {
+        contextPath: nextState.submissionFlow.contextPath,
         submissionId: action.payload.submissionId,
+        subject: nextState.submissionFlow.subject,
+        userId: sd.CURRENT_USER.id,
+        userEmail: sd.CURRENT_USER.email,
```

Comment:
@louislecluse - this event happens on the client and before we were only sending along submission ID. Since we now have additional info around where they came from and the id / email, i figured we should also send that -- cool?